test-objects-chart/templates/daemonset.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: fluentd-elasticsearch
+  namespace: kubelib-tests
```

Comment:
Namespace is defined on chart level, we don't need to hardcode it here.

Reply:
Done.